include/fmt/chrono.h

```diff
@@ -2044,9 +2047,41 @@ class weekday {
 
  public:
   weekday() = default;
-  explicit constexpr weekday(unsigned wd) noexcept
+  explicit constexpr weekday(unsigned int wd) noexcept
       : value(static_cast<unsigned char>(wd != 7 ? wd : 0)) {}
-  constexpr auto c_encoding() const noexcept -> unsigned { return value; }
+  constexpr auto c_encoding() const noexcept -> unsigned int { return value; }
+};
+
+class day {
+ private:
+  unsigned char value;
+
+ public:
+  day() = default;
+  explicit constexpr day(unsigned int d) noexcept
```

Comment:
nit: let's use consistent order of `constexpr` and `explicit`, i.e. change `explicit constexpr` to `constexpr explicit` here and in other places.